Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,103 @@
+OEM Component Subsystem- Auto Light
+------
+
+## The function of the Auto Light Subsystem
+The Master microcontroller will receive the output of the Nissan Leaf AutoLight in this part of the subsystem. Furthermore, the Master microcontroller will detect if the headlight is off, on, blinking, and high and low beams.
+
+### Specification & constraints
+   - Must detect high beam,low beam, left indicator, and right indicator
```

Comment:
What are the voltage and current requirements associated with this? Make this a bit more specific.